app/main/user-content-service.js

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
Suggestion: `s/user-content-service/content-service`

Not all content pages are tailored specifically to the user, and seems like it could be confused with user-agent-service